### DISCUSSION TITLE
Multi v1.3.4.0

### DISCUSSION THREAD

**Discussion by Mickn0488** [pascallanger/DIY-Multiprotocol-TX-Module#957] [General]
Not sure what is going on but, opened companion 2.9.4 this morning and received notification of update for Multiprotocol.
When asked if I want to process hit YES.

 Processing updates for: Multiprotocol
No assets found in release 'Multiprotocol v1.3.4.0' using filter '^MultiLuaScripts'
Multiprotocol preparation failed

Tried manually through but when trying to load does not recognise file.

Just wondering if this is an issue for others and cannot find any further info.

Thanks for great work

**Comment by pascallanger** [comment]
This is most likely due to the fact that the build has been split between AIR and SFC. Companion needs to be updated.
You can download and install v1.3.4.0 manually without an issue from this GitHub.

**Comment by Mickn0488** [comment]
Thanks. For the prompt reply. I tried manually but radio told me wrong format. I’ll try again later when I have woken up properly and let you know. Thanks for the great work. Mick's mobile communicationOn 19 Mar 2024, at 08:40, pascallanger ***@***.***> wrote:﻿
This is most likely due to the fact that the build has been split between AIR and SFC. Companion needs to be updated.
You can download and install v1.3.4.0 manually without an issue from this GitHub.

**Comment by pascallanger** [comment]
Unzip the file that you need most likely mm-stm-serial-aetr-air-v1.3.4.0.bin
On some radio you have to shorten the file name so rename it to something like mm-v1.3.4.0.bin, put this file on the SDCard, use the radio sdcard browser, select the file and flash.
